stdlib.sh

```diff
@@ -1278,6 +1278,27 @@ use_flake() {
   eval "$(nix print-dev-env --profile "$(direnv_layout_dir)/flake-profile" "$@")"
 }
 
+# Usage: use_flakepath [<installable>]
+#
+# Load PATH from your flake, similiar to `nix develop` but without loading a new shell.
+#
+# Behaves like use_flake except that it only modifies your PATH
+#
+# Note that the flakes feature is hidden behind an experimental flag, which
+# you will have to enable on your own. Flakes is not considered stable yet.
+use_flakepath() {
+  watch_file flake.nix
+  watch_file flake.lock
+
+  OLDIFS=$IFS
+  IFS=":"
+
+  for path in $(nix print-dev-env "$@" --json | jq ".variables.PATH.value" -r); do
+    PATH_add $path
+  done
+  export IFS=$OLDIFS
```

Comment:
This would remove the dependency on jq:
```suggestion
  PATH_add "$(nix develop -c sh -c 'echo $PATH')"`
```

Reply:
Copilot suggested me this:

```export PATH=$(nix develop . --command printenv PATH)``` However that'd require people to have printenv (which is part of coreutils). Considering it's this simple when direnv takes care of the PATH diffing I'm not sure if it fits better as a footnote in the docs somewhere rather than a "first class function".

Thoughts? 😄

Reply:
Better to use `sh -c 'echo $PATH'` because `/bin/sh` is on every Unix machine